index.html

```diff
@@ -40,5 +40,209 @@ <h1>I'm Benjamin Glad to see you!</h1>
+              <li>2015</li>
+            </ul>
+          </div>
+        </div>
+
+        <div class="text-area">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </div>
+      </div>
+
+      <div class="middle-section-nav-bars">
+        <ul class="technology">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+      </div>
+
+      <div class="project-link">
+        <h1>See Project</h1>
+      </div>
+  </section>
+
+  
+  <section class="middle-section-container">
+    <div class="image-area">
+      <img src="photo/logos/Snapshoot-Portfolio3.png" alt="">
+    </div>
+
+  <div class="hero">
+      <div class="tonic">
+          <h1>Tonic</h1>
+        <div class="canopy-wrapper">
+          <p>CANOPY</p>
+          <ul>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="text-area">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+      </div>
+    </div>
+
+    <div class="middle-section-nav-bars">
+      <ul class="technology">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+    </div>
+
+    <div class="project-link">
+      <h1>See Project</h1>
+    </div>
+</section>
+
+
+<section class="middle-section-container">
+  <div class="image-area">
+    <img src="photo/logos/Snapshoot-Portfolio4.png" alt="">
+  </div>
+
+<div class="hero">
+    <div class="tonic">
+        <h1>Tonic</h1>
+      <div class="canopy-wrapper">
+        <p>CANOPY</p>
+        <ul>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="text-area">
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+    </div>
+  </div>
+
+  <div class="middle-section-nav-bars">
+    <ul class="technology">
+      <li>html</li>
+      <li>css</li>
+      <li>javascript</li>
+    </ul>
+  </div>
+
+  <div class="project-link">
+    <h1>See Project</h1>
+  </div>
+</section>
+
+<section class="about-myself">
+  <h1 class="header">
+    About Myself
+  </h1>
+
+  <div class="header-text">
+    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+  </div>
+
+  <div class="about-myself-link">
+    <p>LET'S CONNECT</p>
+  </div>
+
+  <div class="about-myself-icons">
+    <img src="photo/logos/github.jpg" alt="">
+    <img src="photo/navs/Linkedin-icon.jpg" alt="">
+    <img src="photo/logos/angellist.jpg" alt="">
+    <img src="photo/logos/twitter.jpg" alt="">
+    <img src="photo/logos/medium.jpg" alt="">
+  </div>
+
+  <div class="resume">
+    <p>Get my resume</p>
+  </div>
```

Comment:
-  [ ] Great job so far but  As I requested above please Fix the margin and padding between the elements and also I can see the font family, font-weight, and color are not the same so please Kindly fix them to match the design👌🏻👌🏻💯💯

- [ ] Please the same for  the border Kindly remove it👌🏻
- your design
![image](https://user-images.githubusercontent.com/58553711/177668740-571f3a2a-a2f0-4cc4-8124-836a1fa484cc.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/177668810-9ecc5dfc-999a-487f-8616-c82f35849174.png)